pvp_club/init.lua

```diff
@@ -78,7 +78,8 @@ minetest.register_on_joinplayer(function(player, n)
                     text = member
                 }
                 if is_owner then
-                    props.text = props.text.." (Owner)"
+                    props.text = props.text..
+                   mt.colorize("#da1818", " (Owner)")
```

Comment:
The line isn't too long so it doesn't need to be on 2 lines.
`lua
props.text = props.text .. mt.colorize("#da1818", " (Owner)")
`

Reply:
Huh?

Reply:
You can use it like this:
```lua
props.text = props.text..mt.colorize("#da1818", " (Owner)")
```
Using 2 lines is unneeded.

Reply:
Ok